TelCo.ColorCoder/Entity.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class Entity
```

Comment:
Which Entity does this represent? Be more specific in naming this class.

Reply:
Done